ruby/lib/minitest/queue/junit_reporter.rb

```diff
@@ -59,7 +59,7 @@ def xml_message_for(test)
         xml = XmlMarkup.new(indent: 2, margin: 2)
         error = test.failure
 
-        if test.skipped?
+        if test.skipped? && !test.flaked?
```

Comment:
Since the junit reporter is flaky test aware, it shouldn't mix actual skips and flakes.